### PR TITLE
2238/features view the wish listed books

### PR DESCRIPTION
**Pull request by gulshan81**
Targets #2238
### Description
 Now we see the wishlisted books 

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: app/Http/Controllers/KnowledgeCafe/Library/BookController.php [hunk 1]
```diff
@@ -27,8 +27,12 @@ public function index()
     {
         $this->authorize('list', Book::class);
         $searchString = (request()->has('search')) ? request()->input('search') : false;
-        $books = Book::getList($searchString);
         $categories = BookCategory::orderBy('name')->get();
+        if (request()->has('show')) {
```
Comment (gautam071) [inline app/Http/Controllers/KnowledgeCafe/Library/BookController.php:31]: Can we rename the `Show` to something related to the issue or fix

File: resources/views/knowledgecafe/library/menu.blade.php [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <ul class="nav nav-pills">
-    <li class="nav-item">
-    <a class="nav-item nav-link {{ $active === 'books' ? 'active' : '' }}" href="{{ route('books.index') }}"><i class="fa fa-book"></i>&nbsp;Books</a>
+    @php
+     $request['books'] = 'books';
```
Comment (gautam071) [inline resources/views/knowledgecafe/library/menu.blade.php:3]: Can we rename the `Books` to something related to the issue or fix

File: resources/views/knowledgecafe/library/menu.blade.php [hunk 3]
```diff
@@ -12,4 +14,10 @@
     <li class="nav-item">
         <a class="nav-item nav-link {{ $active === 'book_a_month' ? 'active' : '' }}" href="{{ route('book.book-a-month.index') }}"><i class="fa fa-book"></i>&nbsp;A Book A Month</a>
     </li>
+    <li class="nav-item">
+        @php
+         $request['show'] = 'booksInWishlist';
```
Comment (gautam071) [inline resources/views/knowledgecafe/library/menu.blade.php:19]: same here

**Review by gautam071 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/KnowledgeCafe/Library/BookController.php (hunk 1 above)
Comment (gulshan81) [reply app/Http/Controllers/KnowledgeCafe/Library/BookController.php:31]: can you please suggest me something?

File: resources/views/knowledgecafe/library/menu.blade.php [hunk 4]
```diff
@@ -12,4 +14,10 @@
     <li class="nav-item">
         <a class="nav-item nav-link {{ $active === 'book_a_month' ? 'active' : '' }}" href="{{ route('book.book-a-month.index') }}"><i class="fa fa-book"></i>&nbsp;A Book A Month</a>
     </li>
+    <li class="nav-item">
+        @php
+         $request['wishlist'] = 'booksInWishlist';
+        @endphp
+        <a class="nav-item nav-link {{ (request()->input('wishlist','active') === 'booksInWishlist') ? 'active' : '' }}"  href="{{ route('books.index', $request)  }}"><i class="fa fa-book"></i>&nbsp;Wishlist-Books</a>
```
Comment (P4NK4J) [inline resources/views/knowledgecafe/library/menu.blade.php:21]: @gulshan81 Do you think a navigation item on UI would have `-` separated titles ?

**Review by P4NK4J — COMMENTED** [review]
left a comment

File: resources/views/knowledgecafe/library/menu.blade.php (hunk 4 above)
Comment (gulshan81) [reply resources/views/knowledgecafe/library/menu.blade.php]: good afternoon @P4NK4J sorry for this late reply. I think you are right the navigation item on UI would not have the - separated titles. thank you.

**Review by gautam071 — APPROVED** [review]
Looks good!

**Pull request merged**